microya/src/androidTest/java/com/papershift/microya/supporting/UploadImageRequest.kt

```diff
@@ -0,0 +1,9 @@
+package com.papershift.microya.supporting
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class UploadImageRequest(
+    val title: String,
+    val description: String,
+)
```

Comment:
Why is there no data to be uploaded in the request? I'm not very familiar with multi-part upload, but I'd expect the request to accept my data to be sent as part of its parameters ... 🤷‍♂️